src/server/services/image.service.ts

```diff
@@ -902,14 +910,15 @@ export const getAllImages = async ({
       u."deletedAt",
       p."availability",
       ${Prisma.raw(
-        includeBaseModel
+        baseModels?.length
```

Comment:
I believe we are using that flag here https://github.com/civitai/civitai/blob/b1286b8fff82e251a50a86927d14ef69d43e9094/src/server/services/collection.service.ts#L872-L884 so I wouldn't make this change